services/dynamic_addressbook_update_node.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.consensus">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+import "google/protobuf/wrappers.proto";
+import "basic_types.proto";
+import "duration.proto";
+import "timestamp.proto";
+
+/**
+ * All fields left null will not be updated.
+ * See [DynamicAddressBookService.updateTopic()](#proto.DynamicAddressBookService)
+ */
+message DynamicAddressBookUpdateNodeTransactionBody {
+    /**
+     * The unique entity id of the DynamicAddress.
+     */
+    int64 nodeID = 1;
+    /**
+     * If set, the new account_id to be associated with the node
+     */
+    AccountID accountID = 2;
+    /**
+     * If set, the new description to be associated with the node
+     */
+    string description = 3;
```

Comment:
For updates, need to use `StringValue`, `Int32Value`, etc wrappers like existing APIs to detect unset vs clear

Reply:
Updated with google.protobuf.* type.